src/Consolonia.Gallery/Consolonia.Gallery.csproj

```diff
@@ -18,5 +18,18 @@
         <PackageReference Include="Avalonia.ReactiveUI" Version="$(AvaloniaVersion)" />
     </ItemGroup>
 
+    <ItemGroup>
+      <AvaloniaXaml Update="Gallery\GalleryViews\GalleryPlatform.axaml">
```

Comment:
I think change GalleryPlatform to mask *  here. So all the pages in gallery will have type Designer and below all will be "DependentUpon"